src/pages/header.tsx

```diff
@@ -0,0 +1,7 @@
+const Header = () => (
+  <div className="absolute z-10">
```

Comment:
Ma really wondering a lot why you are redoing most things when they already existed 
This is not good practice

Reply:
- I have conflicts when pulling from upstream and had to accept incoming changes